dll/win32/shell32/wine/shellord.c

```diff
@@ -279,6 +279,11 @@ VOID WINAPI SHGetSetSettings(LPSHELLSTATE lpss, DWORD dwMask, BOOL bSet)
     if (bSet)
     {
         DWORD changed = 0;
+        if (dwMask & ~g_CachedSSF)
+        {
+            SHELLSTATE tempstate;
+            SHGetSetSettings(&tempstate, dwMask, FALSE); // Read entries that are not in g_CachedSSF
```

Comment:
Is this going to recursively call SHGetSetSettings?

Reply:
It calls itself once yes, but for reading (from the writing code path).

Reply:
ok thanks